Whiskey/Functions/Uninstall-WhiskeyNuGetPackage.ps1

```diff
@@ -25,18 +27,22 @@ function Uninstall-WhiskeyNuGetPackage
         # The version of the package to uninstall. Must be a three part number, i.e. it must have a MAJOR, MINOR, and BUILD number.
         [String]$Version
     )
-        $nugetPath = Join-Path -Path $whiskeyBinPath -ChildPath '\NuGet.exe' -Resolve
-        $Version = Resolve-WhiskeyNuGetPackageVersion -NuGetPackageName $Name -Version $Version -NugetPath $nugetPath
+        Set-StrictMode -Version 'Latest'
+        Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
         if( -not $Version )
         {
-            return
+            $Version = '*'
         }
+
         $packagesRoot = Join-Path -Path $BuildRoot -ChildPath 'packages'
-        $nuGetRootName = '{0}.{1}' -f $Name,$Version
-        $nuGetRoot = Join-Path -Path $packagesRoot -ChildPath $nuGetRootName
-        
-        if( (Test-Path -Path $nuGetRoot -PathType Container) )
+        $packageRoot = Join-Path -Path $packagesRoot -ChildPath ('{0}.{1}' -f $Name,$Version)
+        if( (Test-Path -Path $packageRoot -PathType Container) )
+        {
+            Get-Item -Path $packageRoot | Remove-WhiskeyFileSystemItem
+        }
+        else

```

Comment:
Remove this block. If a package doesn't exist, assume it's already been uninstalled and happily return without doing any work.